tests/other/metadce/hello_libcxx_O2_fexceptions.funcs

```diff
@@ -654,14 +662,20 @@ $std::__2::moneypunct<char\2c\20false>::do_negative_sign\28\29\20const
 $std::__2::moneypunct<char\2c\20false>::do_pos_format\28\29\20const
 $std::__2::moneypunct<char\2c\20false>::moneypunct\28unsigned\20long\29
 $std::__2::moneypunct<char\2c\20false>\20const&\20std::__2::use_facet<std::__2::moneypunct<char\2c\20false>\20>\28std::__2::locale\20const&\29
+$std::__2::moneypunct<char\2c\20true>&\20std::__2::\28anonymous\20namespace\29::make<std::__2::moneypunct<char\2c\20true>\2c\20unsigned\20int>\28unsigned\20int\29
 $std::__2::moneypunct<char\2c\20true>::frac_digits\28\29\20const
+$std::__2::moneypunct<char\2c\20true>::moneypunct\28unsigned\20long\29
 $std::__2::moneypunct<char\2c\20true>::neg_format\28\29\20const
 $std::__2::moneypunct<char\2c\20true>::negative_sign\28\29\20const
 $std::__2::moneypunct<char\2c\20true>::pos_format\28\29\20const
 $std::__2::moneypunct<char\2c\20true>\20const&\20std::__2::use_facet<std::__2::moneypunct<char\2c\20true>\20>\28std::__2::locale\20const&\29
+$std::__2::moneypunct<wchar_t\2c\20false>&\20std::__2::\28anonymous\20namespace\29::make<std::__2::moneypunct<wchar_t\2c\20false>\2c\20unsigned\20int>\28unsigned\20int\29
 $std::__2::moneypunct<wchar_t\2c\20false>::do_decimal_point\28\29\20const
 $std::__2::moneypunct<wchar_t\2c\20false>::do_negative_sign\28\29\20const
+$std::__2::moneypunct<wchar_t\2c\20false>::moneypunct\28unsigned\20long\29
 $std::__2::moneypunct<wchar_t\2c\20false>\20const&\20std::__2::use_facet<std::__2::moneypunct<wchar_t\2c\20false>\20>\28std::__2::locale\20const&\29
+$std::__2::moneypunct<wchar_t\2c\20true>&\20std::__2::\28anonymous\20namespace\29::make<std::__2::moneypunct<wchar_t\2c\20true>\2c\20unsigned\20int>\28unsigned\20int\29
+$std::__2::moneypunct<wchar_t\2c\20true>::moneypunct\28unsigned\20long\29
```

Comment:
this doesn't look right. I think you need to use latest tot LLVM, then this will vanish.

if not, then something else is badly wrong here that we need to figure out.